util/hookable/hookable.cpp

```diff
@@ -78,6 +82,28 @@ static J9CONST_TABLE J9HookInterface hookFunctionTable = {
 #define HOOK_INVALID_ID(id) ((id) | 1)
 #define HOOK_VALID_ID(id) ( (((id) | 1) + 1) )
 
+
+intptr_t
+omrhook_lib_control(const char *key, uintptr_t value)
+{
+	intptr_t rc = -1;
+
+	if (0 != value) {
+#if defined(OMR_RAS_TDF_TRACE)
```

Comment:
With the way this is written, the function will fail 100% of the time if `OMR_RAS_TDF_TRACE` is not defined.